dev/com.ibm.ws.microprofile.openapi.ui/resources/WEB-INF/web.xml

```diff
@@ -26,5 +26,12 @@
     <filter-name>headers-filter</filter-name>
     <url-pattern>/*</url-pattern>
   </filter-mapping>
-  
+  <filter>
+    <filter-name>openapi-ui-filter</filter-name>
+    <filter-class>com.ibm.ws.microprofile.openapi.servlet.filter.OpenAPIEndpointFilter</filter-class>
+  </filter>
```

Comment:
I'm surprised this works, I'm not sure how this bundle can see the `OpenAPIEndpointFilter` class